### PR TITLE
linux-yocto: Adds bluetooth configuration for all platforms

### PR DESCRIPTION
**Pull request by frznlogic**
Signed-off-by: Oscar Andreasson <oscar.andreasson@pelagicore.com>

### CODE REVIEW

**Comment by gunnarx** [comment]
Sorry Oscar.  45537cc4 was a bad merge, hotfixed by reverting in aa7d0132 on master.  I rebased this PR on top of that and pushed it back again.

**Comment by gunnarx** [comment]
Looks like QEMU build failed during the setup stage on some spurious error, maybe a broken connection or some kind of timeout during git submodule update.  I restarted it, but I'd guess this is pretty safe to merge.

File: meta-genivi-dev/meta-genivi-dev/recipes-kernel/linux/linux-yocto_%.bbappend [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
 
-SRC_URI_append_qemux86-64= "\
+SRC_URI_append_qemux86-64 = "\
         file://drm.cfg \
 "
+
+SRC_URI_append += "file://bluetooth.cfg"
```
Comment (DunderRoffe) [inline meta-genivi-dev/meta-genivi-dev/recipes-kernel/linux/linux-yocto_%.bbappend]: the append path needs to start with a space

Comment (frznlogic) [inline meta-genivi-dev/meta-genivi-dev/recipes-kernel/linux/linux-yocto_%.bbappend]: Fixed in the follow up commit

**Comment by gunnarx** [comment]
I don't know the details of the rfkill config you added but don't see any generic problems +1

**Pull request merged**